test/e2e/cloud-provider-oci/load_balancer.go

```diff
@@ -56,7 +56,7 @@ var _ = Describe("Service [Slow]", func() {
 			},
 		},
 	}
-	Context("[cloudprovider][ccm][lb][SL]", func() {
+	Context("[cloudprovider][ccm][lb][SL][system-tags][test]", func() {
```

Comment:
[test] - test related changes should be removed

Reply:
Removed the extra focus